BootstrapKit/Charter.md

```diff
@@ -0,0 +1,28 @@
+## Working Group Charter Template
+
+Every Working Group requires a charter that describes the scope of
+the WGs responsibilities. Once written, a new Pull Request that adds
+the charter to either the [CTC][] or [TSC][] WORKING_GROUPS.md document
+must be opened. The WG is considered to be officially chartered once the
+PR adding the proposed charter is landed and merged into the relevant
+WORKING_GROUPS.md document
+
+### Template: *[insert WG name]* Working Group
+
+The Node.js *[insert WG name]* is jointly governed by a Working Group (WG)
```

Comment:
Ah I see, that makes sense. Perhaps we could go with something like "The Node.js *[insert WG name]* is jointly governed by the members of the WG" to make that more clear?